.circleci/config.yml

```diff
@@ -209,7 +209,7 @@ jobs:
             mvn -Dakkaserverless-sdk.version=$SDK_VERSION test-compile
             mvn -Dakkaserverless-sdk.version=$SDK_VERSION verify -Pit
             echo "==== Verifying that generated unmanaged sources compile ===="
-            rm -rf src/main/java src/test/java
+            rm -rf src/main/java src/test/java src/it/java
```

Comment:
I had to test it myself, but `rm -rf` won't fail if dir doesn't exist. Therefore I'm adding it everywhere. Possible that some of the samples don't have integration tests, but if they get one day, we will be covered.